decompy/RL/LLVMSummary/LLVMSummary.py

```diff
@@ -44,3 +44,4 @@ def __hash__(self):
         :return: hash int
         :rtype: int
         """
+        return hash((self.instcounts, self.loops, self.aa, self.regions))
```

Comment:
Please don't just use hash on a tuple. It is highly likely to get collisions which could break the model. Here is a simple example of what I mean:
>>> hash( (1,0,0) )
2528505496374819208
>>> hash( (1,-2,-2) )
2528505496374819208